docs/tutorial/force_calibration.rst

```diff
@@ -104,3 +118,89 @@ We can plot the calibration by calling::
     calibration.plot()
 
 .. image:: force_calibration_fit.png
+
+Hydrodynamics
+-------------
+
+While the idealized Lorentzian model discussed in the previous section provides a reasonable first description of the physical power spectrum, better models for this spectrum exist.
+
+The idealized model is based on the assumption that the drag force is only proportional to the bead's velocity.
+This assumption is realistic when the bead moves at a constant speed with respect to the fluid.
+When oscillating a bead however, this frictional force becomes frequency dependent.
+
+Equations have been derived by other authors which provide frequency dependent drag coefficients :cite:`tolic2006calibration`.
+These equations can be used to fit a hydrodynamically correct power spectrum:
+
+.. math::
+
+    P_{hydro}(f) = \frac{D Re(\gamma / \gamma_0)}{\pi^2 \left(\left(f_{c,0} + f Im(\gamma/\gamma_0) - f^2/f_{m, 0}\right)^2 + \left(f Re(\gamma / \gamma_0)\right)^2\right)}
+
+where
```

Comment:
where the cutoff/corner frequency is given by

Reply:
👍